Makefile

```diff
@@ -123,6 +129,43 @@ kind-load-test-artifacts: $(KIND) ## Load the e2e testdata container images into
 	$(KIND) load docker-image localhost/testdata/bundles/plain-v0/plain:v0.1.0 --name $(KIND_CLUSTER_NAME)
 	$(KIND) load docker-image localhost/testdata/catalogs/test-catalog:e2e --name $(KIND_CLUSTER_NAME)
 
+.PHONY: opm
+OPM = ./bin/opm
+OPM_VERSION = v1.28.0
+opm: ## Download opm locally if necessary.
+ifeq (,$(wildcard $(OPM)))
+ifeq (,$(shell which opm 2>/dev/null))
+	@{ \
+	set -e ;\
+	mkdir -p $(dir $(OPM)) ;\
+	OS=$(shell go env GOOS) && ARCH=$(shell go env GOARCH) && \
+	curl -sSLo $(OPM) https://github.com/operator-framework/operator-registry/releases/download/$(OPM_VERSION)/$${OS}-$${ARCH}-opm ;\
+	chmod +x $(OPM) ;\
+	}
+else
+OPM = $(shell which opm)
+endif
+endif
+
+.PHONY: install-operator-sdk
+OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/download/v1.30.0
+export OS=$(shell go env GOOS)
+export ARCH=$(shell go env GOARCH) 
+install-operator-sdk:  ## Install operator-sdk
+	if [ ! -f ~/bin/operator-sdk ]; then \
+		curl -LO $(OPERATOR_SDK_DL_URL)/operator-sdk_${OS}_${ARCH} ; \
+		chmod +x operator-sdk_${OS}_${ARCH} ; \
+		mv operator-sdk_$(OS)_$(ARCH) /usr/local/bin/operator-sdk ; \
+	fi
+
+.PHONY: deploy-local-registry
+deploy-local-registry: ## Deploy local docker registry
+	docker run -d -p 5000:5000 --restart=always --name local-registry registry:2
```

Comment:
Our makefile isn't doing this consistently yet but we should probably start using `$(CONTAINER_RUNTIME)` here instead of `docker`.